packages/main/src/SegmentedButton.js

```diff
@@ -213,7 +218,25 @@ class SegmentedButton extends UI5Element {
 			this._selectItem(event);
 		} else if (isSpace(event)) {
 			event.preventDefault();
+		} else if (isPageUp(event)) {
+			event.preventDefault();
+			this._onPageUpOrDown(isPageUp(event));
```

Comment:
if you are already in this "if" block I think isPageUp(event) in "this._onPageUpOrDown(isPageUp(event))" is always true as already checked "else if (isPageUp(event)"

Reply:
Done